extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -848,6 +855,7 @@ export class AmpA4A extends AMP.BaseElement {
     this.adPromise_ = null;
```

Comment:
We don't have to address in this PR but please file an issue for yourself that we should not be removing all children as it will remove placeholder, sizer, and possibly other children.  Instead we should only remove iframe and analytics